plugins/processors/scale/README.md

```diff
@@ -0,0 +1,57 @@
+# Scale Processor Plugin
+
+The scale processor filters for a set of fields,
+and scales the respective values from an input range into
+the given output range according to this formula:
+
+$$\textnormal{result}=(\textnormal{value}-\textnormal{input\_minimum})\cdot
+\frac{(\textnormal{\textnormal{output\_maximum}}-\textnormal{output\_minimum})}
+{(\textnormal{input\_maximum}-\textnormal{input\_minimum})} +
+\textnormal{output\_minimum}$$
+
+The input fields are expected to be numeric.
+Strings representing a single numer are also allowed.
```

Comment:
```suggestion
Input fields are converted to floating point values. If the conversion fails, those fields are ignored.
```
Please check if the second sentence is correct!? You should mention what happens if you have e.g. a string field that contains a text like `foobar`.